yellowbrick/features/base.py

```diff
@@ -293,17 +403,67 @@ def _determine_target_color_type(self, y):
         if y is None:
             self._target_color_type = TargetType.SINGLE
         elif self.target_type == TargetType.AUTO:
-            # NOTE: See #73 for a generalization to use when implemented
```

Comment:
Getting rid of even more technical debt!

Reply:
🙂